indy_node/server/domain_req_handler.py

```diff
@@ -269,13 +270,27 @@ def _validate_revoc_reg_def(self, req: Request):
         assert revoc_def_tag
         assert revoc_def_type
         tags = cred_def_id.split(":")
+
+        cred_def, _, _, _ = self.lookup(cred_def_id, isCommitted=False, with_proof=False)
```

Comment:
Whether this is an EDIT or ADD action for RevocRegDef is defined by a presence of REVOC_DEF, not CRED_DEF

Reply:
Thank you. I did not see that I will update

Reply:
@ashcherbakov When I do this `
        revoc_def, _, _, _ = self.lookup(req.operation[REVOC_REG_DEF_ID], isCommitted=False, with_proof=False)` I get a PoolLedgerTimeout

Reply:
I think this is because there is no `REVOC_REG_DEF_ID` in `req.operation`? I think you need to create `REVOC_REG_DEF_ID` from req data. Have a look at `make_state_path_for_revoc_def`.